### PR TITLE
Ash story

### PR DESCRIPTION
**Pull request by adamShamlian**
@casey-collab

### CODE REVIEW

File: english/matrix/matrix.md [hunk 1]
```diff
@@ -25,3 +25,5 @@ While you don't believe that you might be the One,
 
 [You crawl through the hole you just made and...] (matrix2/matrix2.md)
 
+[There is no darkness - or fluffiness - without the Light.] (../matrix2inception/matrix2inception.md)
```
Comment (casey-collab) [inline english/matrix/matrix.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/matrix/matrix2inception/matrix2inception.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Darkness. Fluffiness.
```
Comment (casey-collab) [inline english/matrix/matrix2inception/matrix2inception.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by adamShamlian** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Looks great! :purple_heart:

**Pull request merged**